### PR TITLE
feat(forms/CollapsibleFieldSet): allow definition of fallback title

### PR DESCRIPTION
**Pull request by sgendre**
**What is the problem this PR is trying to solve?**
TPD had his own implementation of title function, which had a hard coded fallback value: when computed title is empty text,  "Undefined" was returned.

**What is the chosen solution to this problem?**
Allow a fallback value through form definition or widget creation.

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/forms/src/UIForm/fieldsets/CollapsibleFieldset/CollapsibleFieldset.component.js [hunk 1]
```diff
@@ -55,6 +55,10 @@ export function defaultTitle(formData, schema, options) {
 	if (title.length > 0) {
 		return title.join(options?.separator || schema.options?.separator || ', ');
 	}
+
+	if (options?.emptyTitleFallback) {
+		return options.emptyTitleFallback || schema.options?.emptyTitleFallback;
```
Comment (romainseb) [inline packages/forms/src/UIForm/fieldsets/CollapsibleFieldset/CollapsibleFieldset.component.js:60]: i think `options.emptyTitleFallback` will always be return with the current condition. Not sure this is intended

**Review by tlnd-mhuchet — APPROVED** [review]
Have you a lead to understand why the linter is crazy in the CI?

**Review by jmfrancois — APPROVED** [review]
really strange error in the lint if you can find the source it would be great

**Comment by romainseb** [comment]
A fix for the linter issue is there : https://github.com/Talend/ui/pull/3192
The issue here is the fact that there is no lint anymore in this package and at some point (some commit before) the file has been updated this way cause there were some missing bracket or something like that

**Pull request merged**